Potatso/AppInitializer.swift

```diff
@@ -36,15 +36,15 @@ class AppInitializer: NSObject, AppLifeCycleProtocol {
         fileLogger?.logFileManager.maximumNumberOfLogFiles = 7
         DDLog.add(fileLogger)
 
-//        let logglyLogger = LogglyLogger() // Loggy Logger
-//        logglyLogger.logglyKey = "107d98a8-c275-4369-a881-b36d3564c9ef"
-//        let fields = LogglyFields()
-//        fields.userid = User.currentUser.id
-//        fields.appversion = AppEnv.fullVersion
-//        let formatter = LogglyFormatter(logglyFieldsDelegate: fields)
-//        formatter?.alwaysIncludeRawMessage = false
-//        logglyLogger.logFormatter = formatter
-//        DDLog.add(logglyLogger)
+        let logglyLogger = LogglyLogger() // Loggy Logger
+        logglyLogger.logglyKey = "107d98a8-c275-4369-a881-b36d3564c9ef"
```

Comment:
Would you please change back to this `logglyLogger.logglyKey = InfoInternal.shared.getLogglyAPIKey()`?

Reply:
ok.